api/processors/actions/group/abort.py

```diff
@@ -43,7 +43,7 @@ def _set_group_status(self) -> None:
         )
 
     def _set_teachers_status(self) -> None:
-        teachers = Teacher.objects.filter_active()  # type: ignore[attr-defined]
```

Comment:
not sure what happened here, these seem to have been added by a bot and were messing with the mypy check in pre-commit

Reply:
Nah, I don't think bots add those, I think @Roxe322 added them because mypy didn't like the filter. Maybe what happened was that mypy got upgraded at started to understand custom filters, hence now complaining about a redundant `type: ignore`.